samples/gatsby-preview/src/index.jsx

```diff
@@ -0,0 +1,144 @@
+
+    const { contentTypeSlug, isAutoUpdate } = parameters.instance
+
+    this.state = {
+      projectId,
+      projectUrl:
+        "https://" + projectId + "-preview.gtsb.io/" + contentTypeSlug + "/",
+      isAutoUpdate: isAutoUpdate,
+      webhookUrl:
+        "https://backends.ctffns.net/gatsby-preview-proxy/" + projectId
+    }
+  }
+
+  componentDidMount = () => {
+    this.detachFn = this.props.sdk.entry.onSysChanged(this.onSysChanged)
+
+    this.props.sdk.window.startAutoResizer()
+  }
+
+  componentWillUnmount = () => {
+    this.detachFn()
+    if (this.debounceInterval) {
+      clearInterval(this.debounceInterval)
+    }
+  }
+
+  onError = error => {
+    this.setState({ working: false })
+    this.props.sdk.notifier.error(error.message)
+  }
+
+  onSysChanged = () => {
+    if (!this.state.isAutoUpdate) {
+      return
+    }
+    if (this.debounceInterval) {
+      clearInterval(this.debounceInterval)
+    }
+    this.debounceInterval = setInterval(this.refreshGatsbyPreview, 1000)
+  }
+
+  refreshGatsbyPreview = () => {
+    if (this.debounceInterval) {
+      clearInterval(this.debounceInterval)
+    }
+
+    fetch(this.state.webhookUrl, {
+      method: "POST",
+      headers: { "Content-Type": "application/json" },
+      body: JSON.stringify({})
+    }).then(
+      success => this.props.sdk.notifier.success("Gatsby Preview Updated!"),
+      error => this.props.sdk.notifier.error(error)
+    )
+  }
+
+  openPreviewTab = () => {
+    window.open(
+      this.state.projectUrl + this.props.sdk.entry.fields.slug.getValue()
+    )
+  }
+  onChange = e => {
+    this.setState({ isAutoUpdate: e.target.value === "yes" }, () => {
+      console.log(this.state.isAutoUpdate)
+    })
+    //e.target.checked = true
+  }
+
+  render = () => {
+    return (
+      <div className="extension">
+        <div className="flexcontainer">
+          <Forma36.Button buttonType="positive" onClick={this.openPreviewTab}>
+            Open preview
+          </Forma36.Button>
+
+          <Forma36.FieldGroup row={true}>
+            <Forma36.RadioButtonField
+              labelText="Auto-update"
+              disabled={false}
+              checked={this.state.isAutoUpdate}
+              value="yes"
+              onChange={this.onChange}
+              labelIsLight
+              name="autoUpdate"
+            />
+            <Forma36.RadioButtonField
+              labelText="Manual update"
+              disabled={false}
+              checked={!this.state.isAutoUpdate}
+              name="someOption"
+              value="no"
+              onChange={this.onChange}
+              labelIsLight
+              name="autoUpdate"
+            />
+          </Forma36.FieldGroup>
+
+          <Forma36.Button
+            size="small"
+            buttonType="muted"
+            onClick={this.refreshGatsbyPreview}
+          >
+            Update Preview
+          </Forma36.Button>
+
+          <div
+            style={{
```

Comment:
You're mixing inline styles and styles defined in `index.html`. I'd pick one option and stick to it.